webapp/first-snap/python/build.yaml

```diff
@@ -0,0 +1,50 @@
+linux:
+  auto:
+    - action: Return to the root directory of the project containing your snapcraft.yaml and run snapcraft
+      command: snapcraft
+    - action: You will find a .snap file in the same directory that you ran the snapcraft command. You can inspect the contents to ensure it contains all of the application's assets
+      command: unsquashfs -l *.snap
+    - action: Install the snap
+      command: sudo snap install --dangerous *.snap
+    - action: List your installed snaps to confirm
```

Comment:
To confirm what? :P

Reply:
The context in the other steps? :man_shrugging:

Reply:
I know but by reading it it sounds weird